package.json

```diff
@@ -102,13 +101,15 @@
     "eslint-plugin-prettier": "^5.0.1",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "identity-obj-proxy": "^3.0.0",
     "prettier": "3.0.3",
     "resize-observer-polyfill": "^1.5.1",
-    "storybook": "^7.5.1"
+    "storybook": "^7.5.1",
```

Comment:
Storybook is actively broken here, but we'll get back there in the next release.